### DISCUSSION TITLE
LogIn for the Analyst(tm)

### DISCUSSION THREAD

**Discussion by mmoeller86** [devhawala/ST80#4] [Q&A]
Hello dear ST80-Team,

what is the LogIn for the Analyst(tm) ?
User and Password ?

Greets,
Martin

**Comment by devhawala** [comment]
Hello,

see the little table in section [Sample Smalltalk-80 DV6 + Analyst-V1.2 environment](https://github.com/devhawala/ST80#sample-smalltalk-80-dv6--analyst-v12-environment).

The preconfigured users are `analyst`, `demo` and `system`, where the user is also the corresponding password.

Regards
Hans
